en/peripherals/camera.md

```diff
@@ -33,17 +33,24 @@ Mode | Description
 
 ## Trigger Hardware Configuration {#hardware_setup}
 
-You can choose which pins to use for triggering using the [TRIG_PINS](../advanced_config/parameter_reference.md#TRIG_PINS) parameter.
+The pins used to trigger image capture for GPIO, PWM or Seagull-based triggering (i.e. when not using a MAVLink camera) are set using the [TRIG_PINS](../advanced_config/parameter_reference.md#TRIG_PINS) parameter.
 The default is 56, which means that trigger is enabled on *FMU* pins 5 and 6.
 
 > **Note** On a Pixhawk flight controller that has both FMU and I/O boards these FMU pins map to `AUX5` and `AUX6` (e.g. Pixhawk 4, CUAV v5+). 
   On a controller that only has an FMU, the pins map to `MAIN5` and `MAIN6` (e.g. Pixhawk 4 mini, CUAV v5 nano).
   At time of writing triggering only works on FMU pins - you can't trigger a camera using pins on the I/O board.
 
 <span></span>
-> **Warning** With `TRIG_PINS` set to its **default** value of 56, you can use the AUX pins 1, 2, 3 and 4 as actuator outputs (for servos/ESCs).
-  Due to the way the hardware timers are handled (1234 and 56 are 2 different groups handled by 2 timers), this is the ONLY combination which allows the simultaneous usage of camera trigger and FMU actuator outputs.
-  **DO NOT CHANGE THE DEFAULT VALUE OF `TRIG_PINS` IF YOU NEED ACTUATOR OUTPUTS.**
+> **Warning** With `TRIG_PINS=56` (default) you can use the AUX pins 1 to 4 as actuator outputs (for servos/ESCs).
+  With `TRIG_PINS=78`, you can use the AUX pins 1-6 as actuator outputs.
+  Any other combination of pins can be selected, but this will disable use of the other FMU pins as outputs.
+  
+Some cameras can provide image capture signalling to a flight controller camera-capture pin (this enables more accurate image timestamp than the time that the trigger command is sent).
+- Pixhawk FMUv5x boards use the board-specific camera capture pin (PI0).
+- Other board use FMU PWM pin 6 (hardcoded) for camera capture.
+
+> **Note** The camera driver and camera capture driver emit the MAVLink [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.html#CAMERA_TRIGGER) message (with timestamp and image sequence number) when image capture is triggered and signaled, respectively.
+  PX4 determines which instance to use based on the (later) timestamp.
```

Comment:
Where is this coming from?

Reply:
@bkueng This is coming from https://github.com/PX4/Firmware/pull/10295#issuecomment-425932056 . I.e. the intent is clearly that PX4 will determine which value should be used for timestamping, or that another system should be able to work this out from the log.

However I don't truly know if this intent is implemented in code. I would love to be canonical, but I'm not digging in code for this. If you don't either, might this be better "softened" to ...(?)

"A geotagging system can determine which instance to use based on the (later) timestamp (e.g. from the log)."

Reply:
Ah ok, clear now. I would change it then to (or similar):
```
**Note** PX4 emits the MAVLink [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.html#CAMERA_TRIGGER) message (with timestamp and image sequence number).
If camera capture is configured, the timestamp from the camera capture driver is used, otherwise the triggering timestamp.
```

Reply:
Works for me. Changed.